.github/workflows/promote.yml

```diff
@@ -10,6 +10,7 @@ on:
 
 jobs:
   promote:
+    if: github.repository == 'ansible/awx'
```

Comment:
I don't want to lose the ability to test releases out on our personal forks of AWX. This should probably be:

```
endsWith(github.repository, '/awx') 
```

Reply:
```suggestion
    if: endsWith(github.repository, '/awx') 
```